solr/datadog_checks/solr/data/metrics.yaml

```diff
@@ -126,11 +126,29 @@ jmx_metrics:
       category: QUERY
       name: requestTimes
       attribute:
-        avgRequestsPerSecond:
+        MeanRate:
           alias: solr.search_handler.avg_requests_per_sec
```

Comment:
Note: kept `solr.search_handler.avg_requests_per_sec` instead of `solr.search_handler.request_times .avg_requests_per_sec` for backward compatibility